script.js

```diff
@@ -165,4 +175,17 @@ document.addEventListener('DOMContentLoaded', function() {
       seeAllTrigger.parentNode.removeChild(seeAllTrigger);
     });
   }
+
+  // In some cases we should preserve focus after page reload, see #HCJ-1488
+  function saveFocus() {
+    var activeElementId = document.activeElement.getAttribute("id");
+    sessionStorage.setItem('returnFocusTo', '#' + activeElementId);
+  }
```

Comment:
Nit: define  the function before usage to avoid hoisting and for consistency.

Reply:
Thanks for the comments.
Moved it to the top.